### DISCUSSION TITLE
image do not show up anymore after the update

### DISCUSSION THREAD

**Discussion by Masabu** [binokochumolvarghese/lightbi-hugo#53] [Q&A]
Hello!! 

I have updated the lightbt-hugo to the most recent one via github - then pre-view image stopped showing up. Was there any setting I was supposed to do?

my hugo's version is "hugo v0.111.3+extended freebsd/amd64 BuildDate=2023-03-17T03:01:46Z+0000 VendorInfo=freebsd"

Steps I have taken

- git removed lightbi-hugo 
- then add git submodule lightbi-hugo
- copied the most recent config.toml from github - revised based on my environment
- buit page with hugo

https://masahikoaida.com/

**Comment by binokochumolvarghese** [comment]
Hi,
In the latest versin images have been moved to asset folder for adding new features as per latest Hugo. Please move your images to asset folder as shown [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo/tree/main/themes/lightbi-hugo/assets/img).
